run-checks.ts

```diff
@@ -253,7 +253,7 @@ async function main() {
   };
 
   // Generate reports
-  const dir = `./reports/${config.daoName}/${config.governorAddress}`;
+  const dir = `./${REPORTS_BRANCH}/${config.daoName}/${config.governorAddress}`;
```

Comment:
have we overloaded this `REPORTS_BRANCH` variable? in constants.ts, the comment on line 27 implies this variable is only needed for new [simulated] proposals
https://github.com/uniswapfoundation/governance-seatbelt/blob/fc6aa0a26c2c700014e9581fd7f73136aeffdd33/utils/constants.ts#L27-L32

but here we use it as the reports directory

if we're using it as a reports output directory, i'd ask for it to be called `REPORTS_OUTPUT_DIRECTORY` or `REPORTS_DIRECTORY` or `REPORT_PATH` etc.

but then i'd also ask to confirm we don't use it for anything else.